crates/sp-domains/src/verification.rs

```diff
@@ -141,29 +147,42 @@ where
     }
 
     let storage_key = SK::block_randomness_key();
-    let storage_proof = randomness_proof.clone();
     let block_randomness = StorageProofVerifier::<Hashing>::verify_and_get_value::<Randomness>(
         &consensus_state_root,
-        storage_proof,
+        randomness_proof.clone(),
```

Comment:
It was `randomness_storage_proof` in the other place, helpful to keep naming consistent across codebase.